src/Smart.Design.Console/CommandOptions/GenerateOptions.cs

```diff
@@ -0,0 +1,11 @@
+using CommandLine;
+
+namespace Smart.Design.Console.CommandOptions
+{
+    [Verb("generate", HelpText = "Generates html for a specific design element")]
```

Comment:
Please write "HTML" in all caps (just like you would do in a README for instance.

Reply:
OK